components/Frame/Popover/UserNav.js

```diff
@@ -0,0 +1,254 @@
+import React, { useRef, useEffect, useState } from 'react'
+import { compose, graphql } from 'react-apollo'
+import { css } from 'glamor'
+import {
+  colors,
+  fontStyles,
+  mediaQueries,
+  Center,
+  Button,
+  TeaserSectionTitle
+} from '@project-r/styleguide'
+import { HEADER_HEIGHT, HEADER_HEIGHT_MOBILE } from '../../constants'
+
+import withT from '../../../lib/withT'
+import withInNativeApp from '../../../lib/withInNativeApp'
+import { matchPath } from '../../../lib/routes'
+import SignIn from '../../Auth/SignIn'
+import SignOut from '../../Auth/SignOut'
+import { withMembership } from '../../Auth/checkRoles'
+import Footer from '../Footer'
+import NavLink, { NavA } from './NavLink'
+import NotificationFeedMini from '../../Notifications/NotificationFeedMini'
+import Link from '../../Link/Href'
+import BookmarkMiniFeed from '../../Bookmarks/BookmarkMiniFeed'
+
+const SignoutLink = ({ children, ...props }) => (
+  <div {...styles.signout}>
+    <NavA {...props}>{children}</NavA>
+  </div>
+)
+
+const UserNav = ({
+  me,
+  router,
+  expanded,
+  closeHandler,
+  t,
+  inNativeApp,
+  isMember
+}) => {
+  const [containerPadding, setContainerPadding] = useState()
+  const containerRef = useRef(null)
+  useEffect(() => {
+    const measureLeftPadding = () => {
+      if (containerRef.current) {
+        const containerWidth = containerRef.current.clientWidth
+        const windowWidth = window.innerWidth
+        setContainerPadding((windowWidth - containerWidth) / 2)
+      }
+    }
+    window.addEventListener('resize', measureLeftPadding)
+    measureLeftPadding()
+    return () => {
+      window.removeEventListener('resize', measureLeftPadding)
+    }
+  }, [])
+
+  const active = matchPath(router.asPath)
+  const hasExpandedRef = useRef(expanded)
+  if (expanded) {
+    hasExpandedRef.current = true
+  }
+  return (
+    <>
+      <hr {...styles.hr} {...styles.hrFixed} />
+      <Center {...styles.container} id='nav'>
+        <div ref={containerRef}>
+          {hasExpandedRef.current && (
+            <>
+              {!me && (
+                <>
+                  <div {...styles.signInBlock}>
+                    <SignIn style={{ padding: 0 }} />
+                  </div>
+                </>
+              )}
+              {!isMember && (
+                <Button
+                  style={{ marginTop: 24 }}
+                  href='https://www.republik.ch/pledge'
+                  black
+                  block
+                >
+                  {t('nav/becomemember')}
+                </Button>
+              )}
+              {me && (
+                <>
+                  <Link href='/benachrichtigungen' active={active} passHref>
```

Comment:
Please also use the `Link` from `lib/routes` here. And remove the `active` prop – not supported on either of those – that a `NavLink` specific thing.

```suggestion
                  <Link route='subscriptions' passHref>
```